docs/user/openstack/customization.md

```diff
@@ -16,6 +16,9 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 ## Machine pools
 
 * `type` (optional string): The OpenStack flavor name for machines in the pool.
+* `rootVolume` (optional object): Defines the root volume for instances in the machine pool. The instances use ephemeral disks if not set.
+  * `size` (integer): Size of the root volume in GB.
+  * `type` (string): The volume pool to create the volume from.
```

Comment:
These should be "optional integer" and "optional string" (or maybe "required string"?) respectively.  Or maybe they're both required?  Either way, say "optional" or "required", and if optional, say something about the defaults, unless the default is "we'll magically pick something that works".